README.md

```diff
@@ -48,7 +48,7 @@ When speed is essential, applications can selectively copy the external data int
 
 
 ## Downloading and Installing SnappyData
-You can download and install the latest version of SnappyData from [github](https://github.com/SnappyDataInc/snappydata/releases) or you can download the enterprise version that is TIBCO ComputeDB from [here](https://edelivery.tibco.com/storefront/index.ep).
+You can download and install the latest version of SnappyData from [github](https://github.com/TIBCOSoftware/snappydata/releases) or you can download the enterprise version that is TIBCO ComputeDB from [here](https://edelivery.tibco.com/storefront/index.ep).
```

Comment:
Have done similar changes in another branch SDENT-196 which would be merged into master once all changes are completed and the process is approved by Legal.

Reply:
Ok will revert